gensim/test/test_coherencemodel.py

```diff
@@ -44,6 +44,11 @@ def setUp(self):
             ['user', 'graph', 'minors', 'system'],
             ['time', 'graph', 'survey', 'minors']
         ]
+        self.topics3 = [
+            ['human', 'computer', 'system', 'interface'],
+            ['graph', 'minors', 'trees', 'eps']
+        ]
```

Comment:
I think you need at least 2 more tests to cover your functionality:

- Topics are IDs
- Topics are neither IDs nor tokens (check that ValueError is raised)

Reply:
And added the 2 new tests (also updated the one using self.topics3 because I forget to change at least one token with respect to self.topics1 in the last update

Reply:
Is there a reason why this isn't a separate test (or set of tests)? What's the benefit of keeping everything in a single test method?